packages/x-components/src/x-modules/related-tags/components/__tests__/related-tags.spec.ts

```diff
@@ -116,12 +116,12 @@ describe('testing related tags component', () => {
     unSelectedWrappers.forEach(expectToHaveOverriddenContent);
   });
 
```

Comment:
What about the test for each related tag content?

Reply:
I tried but it was really difficult to test.

Reply:
Done!